code/datums/chatmessage.dm

```diff
@@ -492,7 +492,8 @@
 		qdel(src)
 		return
 	//handle color
-	tgt_color = color
+	if(color)
+		tgt_color = color
```

Comment:
Should this var have any value if there's no colour?

Reply:
It'd have the default one for balloon alerts in that case